_maps/map_files/Birdshot/birdshot.dmm

```diff
@@ -25667,10 +25665,6 @@
 "jKU" = (
 /turf/closed/wall,
 /area/station/engineering/atmos/storage/gas)
-"jLi" = (
-/obj/effect/landmark/start/virologist,
-/turf/open/floor/iron/showroomfloor,
-/area/station/medical/virology)
```

Comment:
1

Reply:
This is likely just key reshuffling, unless you verified it in map editor nothing to be worried about